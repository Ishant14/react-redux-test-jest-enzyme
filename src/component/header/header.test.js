import React from 'react';
import Header from './header';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

describe("Header Component ", () => {
    test("it should render properly ", () => {
        const component = renderer.create(<Header/>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })

    test("it should have  ", () => {
        const component = shallow(<Header/>);
        const wrapper = component.find('.wrap');
        expect(wrapper.length).toBe(1);
    })

})