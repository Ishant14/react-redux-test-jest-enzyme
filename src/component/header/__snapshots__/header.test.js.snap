// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component  it should render properly  1`] = `
<header>
  <div
    className="wrap"
  >
    <div
      className="logo1"
    >
      REACT-JEST
    </div>
    <div>
      sdf
    </div>
  </div>
</header>
`;
